mediator/med_merge_mod.F90

```diff
@@ -290,16 +293,18 @@ subroutine med_merge_auto_field(merge_type, FBout, FBoutfld, FB, FBfld, FBw, fld
     end if
 
     ! error checks
-    if (ungriddedUBound_output(1) /= ungriddedUBound_input(1)) then
-       call ESMF_LogWrite(trim(subname)//"ungriddedUBound_input not equal to ungriddedUBound_output", ESMF_LOGMSG_INFO)
-       rc = ESMF_FAILURE
-       return
-    else if (gridToFieldMap_input(1) /= gridToFieldMap_output(1)) then
-       call ESMF_LOGWrite(trim(subname)//"gridToFieldMap_input not equal to gridToFieldMap_output", ESMF_LOGMSG_INFO)
-       rc = ESMF_FAILURE
-       return
-    end if
-
+    if (lrank == 2) then
+       if (ungriddedUBound_output(1) /= ungriddedUBound_input(1)) then
+          write(errmsg,*) trim(subname),"ungriddedUBound_input (",ungriddedUBound_input(1),") not equal to ungriddedUBound_output (",ungriddedUBound_output(1),")"
```

Comment:
Can you please check that this line is not longer than 132 characters - and break it up if it is

Reply:
fixed